locales/en/app.ftl

```diff
@@ -762,9 +762,37 @@ vpn-banner-cta-expand = Expand
 # button to close panel
 vpn-banner-cta-close = Close
 
-## VPN and Relay educational videos
-
-# Relay video headline for placement on Monitor home page
-relay-video-headline = How do you keep your email address a secret?
-# VPN video headline for placement on Monitor user's dashboard
-vpn-video-headline = Do you worry about safety on public Wi-Fi?
+## Relay and VPN educational/ad units
+
+ad-unit-relay-cta = Learn more about { -brand-relay }
+ad-unit-vpn-cta = Learn more about { -brand-mozilla-vpn }
+# ad 1 heading
+ad-unit-1-how-do-you-keep = How do you keep your email address a secret?
+# ad 2 heading
+ad-unit-2-do-you-worry = Do you worry about safety on public Wi-Fi?
+# ad 3 heading
+ad-unit-3-stay-in-the-game = Stay in the game!
+ad-unit-3-lets-you-keep = { -brand-mozilla-vpn } lets you keep a stable connection safe and secure while you play games or stream movies.
+ad-unit-3-prevent-throttling = Prevent throttling
```

Comment:
How are this and the 2 following strings used? There's no context in the ID, and I can't really tell without comments.

Reply:
added comments denoting list items/bullet points